src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/mms/MMSDeltaProjectEventListenerAdapter.java

```diff
@@ -260,10 +260,10 @@ private void determineChangesUsingCommitResponse(File responseFile, Set<String>
                             }
 
                             size = validateJsonElementArray(currentNode, isSyncingCommit, lockedElementIds, size);
-                        }
-                    } else { // what do we throw here? and should we?
-                        throw new IllegalStateException();
-                    }
+//                        }
```

Comment:
Let's remove instead of commenting out and fix the indentation of the nested section.